### ISSUE TITLE
No opencomputer's chat commands

### ISSUE THREAD

**Issue by astral17** [KosmosPrime/OpenComputers#16]
Version 1.16.5 don't have any of the commands available in 1.12.2 version. Some of them were quite useful. For example /oc_sc or /oc_spawnComputer, which spawned an already equipped creative computer. Some were occasionally used for debugging, etc.

**How it should look 1.12.2 (OC 1.8.5):**
![image](https://github.com/KosmosPrime/OpenComputers/assets/20907711/00b05bd2-0c20-4c38-b988-6ba8c48003ee)
**How it looks 1.16.5 (OC 1.8.5.1):**
![image](https://github.com/KosmosPrime/OpenComputers/assets/20907711/53025f9f-e0cc-4b0e-a8f3-9aae614d7125)

**Comment by KosmosPrime** [comment]
Yes, I'd originally deferred implementing the commands because I didn't see them as particularly important. Shortly thereafter OC went into limbo and the port followed, so I never caught up on these.
